docs/cron-workflows.md

```diff
@@ -103,6 +104,33 @@ For example, with timezone set at `America/Los_Angeles`, we have daylight saving
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+### Stopping a `CronWorkflow`
```

Comment:
The cronworkflow is not set as "suspended", it is just removed from the queue and cannot become active again, so I think it is best to be different.

Reply:
Oh word, I misunderstood the feature, you're right, "stop" is the correct wording then.
Huh I would think it would just auto-delete after that so it doesn't take up space in etcd 🤔 (although I can see that keeping it _somewhere_ might be useful for historical/archive/compliance purposes, although you already have the Workflows themselves)